OneCommunityGlobal/HighestGoodNetworkApp
# Zijie lint fix for reports component Phase 4

# Description
Lint fix for Reports component, this is phase 4.

## Files changed

- src/components/Reports/TotalReport/TotalPeopleReport.jsx
- src/components/Reports/TotalReport/TotalProjectReport.jsx
- src/components/Reports/TotalReport/TotalReportBarGraph.jsx
- src/components/Reports/TotalReport/TotalTeamReport.jsx

# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Go to Reports,  Check functionality is the same as before.
- Check dashboard - Report - Total Report is normal.(Please record video)
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record video)

Optional:
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint src/components/Reports/TotalReport/TotalPeopleReport.jsx`
- `npx eslint src/components/Reports/TotalReport/TotalProjectReport.jsx`
- `npx eslint src/components/Reports/TotalReport/TotalReportBarGraph.jsx`
- `npx eslint src/components/Reports/TotalReport/TotalTeamReport.jsx`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!

Screenshot:
<img width="1233" alt="CleanShot 2023-09-30 at 12 03 49@2x" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46791134/526d2a4b-a6fb-430d-b79d-71d0d363c7a5">
